### ISSUE TITLE
Titanium SDK 9.2.2GA - no such table: resources

### ISSUE THREAD

**Issue by drunnells** [Pushwoosh/pushwoosh-appcelerator-titanium#81]
I am receiving the below error message in the logs when running my app on Android 11 (Pixel 5) using Appcelerator Titanium SDK 9.2.2GA and the Pushwoosh Titanium module v6.0.0. The app seems to be functional and is receiving pushes as expected, but this error appears as soon as the app starts:

```
[ERROR] :  SQLiteLog: (1) no such table: resources in "SELECT * FROM resources"
[ERROR] :  Pushwoosh: Can't get cached resources:
[ERROR] :  Pushwoosh: android.database.sqlite.SQLiteException: no such table: resources (code 1 SQLITE_ERROR): , while compiling: SELECT * FROM resources
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1045)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:652)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:61)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:46)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1545)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteDatabase.queryWithFactory(SQLiteDatabase.java:1392)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1263)
[ERROR] :  Pushwoosh:   at android.database.sqlite.SQLiteDatabase.query(SQLiteDatabase.java:1431)
[ERROR] :  Pushwoosh:   at com.pushwoosh.repository.w.a(Unknown Source:17)
[ERROR] :  Pushwoosh:   at com.pushwoosh.notification.handlers.message.user.b$c.a(Unknown Source:4)
[ERROR] :  Pushwoosh:   at com.pushwoosh.notification.handlers.message.user.b$c.doInBackground(Unknown Source:2)
[ERROR] :  Pushwoosh:   at android.os.AsyncTask$3.call(AsyncTask.java:394)
[ERROR] :  Pushwoosh:   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[ERROR] :  Pushwoosh:   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
[ERROR] :  Pushwoosh:   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
[ERROR] :  Pushwoosh:   at java.lang.Thread.run(Thread.java:923)
```

**Comment by wfhm** [comment]
@drunnels,

Checking it.

**Comment by wfhm** [comment]
@drunnels,

Got it reproduced, however, it does not affect SDK's functionality, so this will be fixed in future releases, but we cannot give this issue a high priority right now. I will additionally update you in this thread as soon as the fix is rolled out.

**Comment by wfhm** [comment]
@drunnels,

Fixed in 6.0.1.
